lib/chore/cli.rb

```diff
@@ -266,7 +266,25 @@ def validate! #:nodoc:
         puts @parser
         exit(1)
       end
+    end
 
+    def validate_strategy!
```

Comment:
True - validation of the config is never something we really got quite
right.

On Mon, Apr 18, 2016 at 11:41 AM, Tanner Burson <notifications@github.com>
wrote:

> In lib/chore/cli.rb
> <https://github.com/Tapjoy/chore/pull/31#discussion_r60081736>:
>
> >
> > +    def validate_strategy!
>
> @StabbyCutyou <https://github.com/StabbyCutyou> This is another point
> towards needing to refactor the config support. Clunky that adding a set of
> strategies requires modifying the CLI class to handle validation cleanly.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Tapjoy/chore/pull/31/files/dc6f9a67882f95933bcc40eb8223964b5ae1dde7#r60081736>
>